ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -108,10 +108,23 @@ class SignalProducerSpec: QuickSpec {
 		}
 
 		describe("SignalProducer.buffer") {
-			pending("should replay buffered events when started, then forward events as added") {
+			it("should replay buffered events when started, then forward events as added") {
+				var counter = 0
+				let (producer, sink) = SignalProducer<Int, NSError>.buffer(1)
+				
+				producer.start(Event.sink(next:{value in counter += value}))
+				sendNext(sink, 1)
+				sendNext(sink, 1)
+				
+				expect(counter).to(equal(2))
 			}
 
-			pending("should drop earliest events to maintain the capacity") {
+			it("should drop earliest events to maintain the capacity") {
+				let (producer, sink) = SignalProducer<Int, NSError>.buffer(1)
+				sendNext(sink, 0)
+				sendNext(sink, 1)
+				
+				expect(producer).to(sendValue(1, sendError: nil, complete: false))
```

Comment:
Can you test sending an error _after_ this and make sure that it only forwards that?